app/models/ticket.rb

```diff
@@ -25,6 +25,6 @@ class Ticket < ApplicationRecord
   delegate :name, to: :area, prefix: :area
 
   def schedule_auto_checkout_job
-    AutoCheckoutTicketJob.set(wait: AUTO_CHECKOUT_AFTER).perform_later(id)
+    AutoCheckoutTicketJob.set(wait: company.owner.auto_checkout_minutes.try(:minutes) || AUTO_CHECKOUT_AFTER).perform_later(id)
```

Comment:
```suggestion
    AutoCheckoutTicketJob.set(wait: company.owner.auto_checkout_minutes&.minutes || AUTO_CHECKOUT_AFTER).perform_later(id)
```
Marco always told me not to use try cause it hides too many error conditions :smile: